st2common/st2common/models/api/action.py

```diff
@@ -410,6 +411,18 @@ def to_model(cls, live_action):
         return model
 
 
+class LiveActionCreateAPI(LiveActionAPI):
```

Comment:
See my comment here - https://github.com/StackStorm/st2/pull/2723#issuecomment-222937619

In short, that's the model for input, otherwise is for output.